### PR TITLE
[google-apps-script] Updated Google Classroom API

### PR DESCRIPTION
**Pull request by NathanAdhitya**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<https://developers.google.com/classroom/reference/rest>>
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

Regenerated with the generator specified. Unnecessary "export" was removed, erroring exports were also removed. Done to add the new CourseWorkMaterial typings.

### CODE REVIEW

File: types/google-apps-script/apis/classroom_v1.d.ts [hunk 1]
```diff
@@ -1,1195 +1,706 @@
...
-            // * `NOT_FOUND` if the requested course, course work, or student submission
-            // does not exist.
-            turnIn(resource: any, courseId: string, courseWorkId: string, id: string): void;
-          }
+    namespace Classroom {
+        namespace Collection {
+            namespace Courses {
+                namespace CourseWork {
+                    interface StudentSubmissionsCollection {
+                        // Returns a student submission. * `PERMISSION_DENIED` if the requesting user is not permitted to access the requested course, course work, or student submission or for access errors. * `INVALID_ARGUMENT` if the request is malformed. * `NOT_FOUND` if the requested course, course work, or student submission does not exist.
```
Comment (rbuckton) [inline types/google-apps-script/apis/classroom_v1.d.ts:12]: The format of the comment here seems worse. It was on multiple lines before, now its all on a single line.

**Review by rbuckton — CHANGES_REQUESTED** [review]

File: types/google-apps-script/apis/classroom_v1.d.ts (hunk 1 above)
Comment (NathanAdhitya) [reply types/google-apps-script/apis/classroom_v1.d.ts:12]: Ouch. Didn't notice that. Seems like prettier didn't also clean up comments as all I did was regenerate and remove errors. Will fix.

Comment (NathanAdhitya) [reply types/google-apps-script/apis/classroom_v1.d.ts]: Split the return status comments into multiple lines. Thank you for the input.

**Review by rbuckton — APPROVED** [review]

**Comment by NathanAdhitya** [comment]
Ready to merge

**Pull request merged**